test/testcafe/spec/ChallengeCustomOTPAuthenticatorView_spec.js

```diff
@@ -0,0 +1,63 @@
+import { RequestMock } from 'testcafe';
+import xhrAuthenticatorVerifyCustomOTP from '../../../playground/mocks/data/idp/idx/authenticator-verification-custom-otp';
+import xhrInvalidOTP from '../../../playground/mocks/data/idp/idx/error-authenticator-verification-custom-otp';
+import xhrSuccess from '../../../playground/mocks/data/idp/idx/success';
+import ChallengeCustomOTPPageObject from '../framework/page-objects/ChallengeCustomOTPPageObject';
+import SuccessPageObject from '../framework/page-objects/SuccessPageObject';
+
+const mockChallengeAuthenticatorCustomOTP = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrAuthenticatorVerifyCustomOTP)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+  .respond(xhrSuccess);
+
+const mockInvalidPasscode = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrAuthenticatorVerifyCustomOTP)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+  .respond(xhrInvalidOTP, 403);
+
+fixture('Challenge Authenticator Custom OTP');
+
+async function setup(t) {
```

Comment:
Please also check the event as other authenticators do: https://github.com/okta/okta-signin-widget/blob/master/test/testcafe/spec/EnrollAuthenticatorGoogleAuthenticator_spec.js#L39
Same comment for the enroll test.